src/debug.ts

```diff
@@ -63,15 +63,15 @@ export function getRouteInfo(container: inversifyInterfaces.Container) {
                 }
             }
 
-            const details = {
+            const details: { route: string, args?: string[] } = {
```

Comment:
I'd suggest to create a type por this andset the returning type of `getRouteInfo` to this type